--- a/src/element/text/Text.cpp
+++ b/src/element/text/Text.cpp
@@ -358,7 +358,11 @@
         pango_layout_set_wrap(pangoData.layout, PANGO_WRAP_WORD_CHAR);
         pango_layout_set_ellipsize(pangoData.layout, PANGO_ELLIPSIZE_NONE);
     } else {
+#if PANGO_VERSION_CHECK(1, 56, 0)
         pango_layout_set_wrap(pangoData.layout, PANGO_WRAP_NONE);
+#else
+        pango_layout_set_wrap(pangoData.layout, PANGO_WRAP_WORD);
+#endif
         pango_layout_set_ellipsize(pangoData.layout, PANGO_ELLIPSIZE_END);
     }
 }
